package.json

```diff
@@ -37,25 +40,28 @@
     "@semantic-release/changelog": "^5.0.1",
     "@semantic-release/exec": "^5.0.0",
     "@semantic-release/git": "^9.0.0",
-    "@wordpress/eslint-plugin": "^7.2.1",
-    "@wordpress/scripts": "^12.2.1",
+    "@wordpress/eslint-plugin": "^7.4.0",
+    "@wordpress/scripts": "^12.6.1",
     "conventional-changelog-simple-preset": "^1.0.14",
     "cypress": "^5.3.0",
     "grunt": "^1.2.1",
     "grunt-version": "^2.0.0",
     "grunt-wp-readme-to-markdown": "^2.0.1",
     "npm-run-all": "^4.1.5",
     "replace-in-file": "^6.1.0",
-    "rtlcss": "^2.6.0",
-    "semantic-release": "^17.0.8",
+    "rtlcss": "^2.6.2",
+    "semantic-release": "^17.3.7",
     "whatwg-fetch": "^3.4.1"
   },
   "dependencies": {
-    "@wordpress/icons": "^2.7.0",
+    "@wordpress/icons": "^2.9.0",
     "classnames": "^2.2.6",
-    "fuse.js": "^6.4.1",
-    "query-string": "^6.13.6",
+    "fuse.js": "^6.4.6",
+    "i": "^0.3.6",
+    "npm": "^6.14.11",
```

Comment:
@HardeepAsrani I don't think these two should be here :see_no_evil:

Reply:
Yea, I was resetting somethings and npm added that. I'll remove it. :D